projects/js-packages/social-logos/changelog/add-substack-to-social-logo

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Versioning is a bit ambiguous, but if we call it a minor update, tooling expects the package version to be 3.2.0-alpha. Since it's just one icon addition, let's call this a patch-level and keep things simple.

```suggestion
Significance: patch
```